hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/MonitorServiceRestartParentAttributeHandler.java

```diff
@@ -54,11 +53,6 @@ protected ServiceName getParentServiceName(PathAddress parentAddress) {
     }
 
     @Override
-    protected boolean isResourceServiceRestartAllowed(OperationContext context, ServiceController<?> service) {
```

Comment:
Yes, correct. And for our websocket clients, they will be told about the new state of the server and if they need to tell it to restart or reload via the restart flag in the json response (me and Jay were reminded of this fact yesterday when the itests failed due to this flag being set when we weren't expecting it to be :)

DMR clients will also know this because they, too, are given the reload/restart flag in the response.